models/workload.go

```diff
@@ -13,11 +13,24 @@ import (
 	"github.com/kiali/kiali/config"
 )
 
+type ClusterWorkloads struct {
+	// Cluster where the apps live in
+	// required: true
+	// example: east
+	Cluster string `json:"cluster"`
```

Comment:
Would it make sense to add `Cluster` to `WorkloadList` instead of creating a new model without `Namespace`?

Reply:
Ideally `WorkloadList` should be gone at some point. Because the idea is to tight Workloads into Cluster.